### DISCUSSION TITLE
Industry Vs Commodity 1.2

### DISCUSSION THREAD

**Discussion by writetolalit** [USEPA/supply-chain-factors#55] [Q&A] [answered]
Pardon if this is a very basic question.

V1, V1.1, and V1.1.1 information were published by commodity and industry (detailed and summary) from 2010 to 2016 in Kg GHG per 2018 USD. The data for v1.2 found here (https://catalog.data.gov/dataset/supply-chain-greenhouse-gas-emission-factors-v1-2-by-naics-6/resource/d2b3a25c-f4d2-498d-8df5-ad15e1626250) is only by industry.

Will there be a precompiled data release by commodity for v1.2 in the near future?

**Comment by bl-young** [comment, marked as answer]
The v1.2 release is based on a commodity model. See the [documentation](https://pasteur.epa.gov/uploads/10.23719/1528686/documents/Aboutthe2019v1.2SupplyChainGHGEmissionFactors.pdf) for more details.

See also https://github.com/USEPA/supply-chain-factors/discussions/42

**Reply by writetolalit** [reply]
Ben,

Thanks for the quick response. My confusion stems from the language. 

_For example, in 1.1.1_
USEEIO code for **1111A0** is listed as "Fresh soybeans, canola, flaxseeds, and other oilseeds" for commodity and "Oilseed farming" for Industry. 
_In 1.2_
USEEIO Reference Code **1111A0** is split into two NAICS codes, **111110** "Soybean Farming," and **111120** "Oilseed (except Soybean) Farming". 

The word choice seems similar to industry rather than commodity, so I assumed it was indicating industry.

**Reply by bl-young** [reply]
Yes you are correct, in the earlier versions we updated the sector names to be more consistent as commodities instead of industries.

However, for this NAICS based schema in v1.2, we just use the names as defined in the NAICS schema as published. But the model itself is a Commodity model (see [line 10 of the model spec](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L10))

**Reply by writetolalit** [reply]
Thanks for the clarification.
